bootstrap/bootstrap.go

```diff
@@ -122,9 +122,13 @@ func RunAndReturnWaitGroup(
 			fatalError(fmt.Errorf("failed to create SecretProvider: %s", err.Error()), lc)
 		}
 
-		// Bypass the zero trust zitidfied transport for Core Keeper Configuration client
-		// Should leverage the HttpTransportFromService function from zerotrust pkg, set the default transport for now
-		secretProvider.SetHttpTransport(http.DefaultTransport)
+		bootstrapConfig := serviceConfig.GetBootstrap()
```

Comment:
if the fallback mechanism shown below works, i would expect all these "TODO"/"keeper" related lines to be able to be removed